### PR TITLE
Add an option to pass shfmt arguments

### PR DESCRIPTION
**Pull request by Roang-zero1**
I've added a env variable to pass argument to the shfmt tool as I need to pass the indentation options.

### CODE REVIEW

**Comment by jbergstroem** [comment]
Same here; more specifically `-ln posix`.

File: shfmt/entrypoint.sh [hunk 1]
```diff
@@ -5,12 +5,16 @@ set -euo pipefail
 # shellcheck disable=SC1091
 source /lib.sh
 
+if [ -z "${SHFMT_ARGS:-}" ]; then
+	SHFMT_ARGS=""
+fi
+
 fix() {
-	shfmt -s -w .
+	shfmt -s -w ${SHFMT_ARGS} .
```
Comment (bltavares) [inline shfmt/entrypoint.sh]: What do you think of in-lining it directly? We could avoid the `[ -z ]` test with that, and it's an idiom I'm used to use on `bash`.

```suggestion
	shfmt -s -w ${SHFMT_ARGS:-} .
```

**Review by bltavares — COMMENTED** [review]
Sorry for the delay, I didn't noticed the new notification on this repo.

I've sent you a suggestion on the code, what do you think of the change?

File: shfmt/entrypoint.sh (hunk 1 above)
Comment (Roang-zero1) [reply shfmt/entrypoint.sh]: Yes good suggestions, I've committed it.

File: shfmt/entrypoint.sh [hunk 2]
```diff
@@ -5,12 +5,16 @@ set -euo pipefail
...
+fi
+
 fix() {
-	shfmt -s -w .
+	shfmt -s -w ${SHFMT_ARGS:-} .
 }
 
 lint() {
-	shfmt -s -l -d .
+	shfmt -s -l -d ${SHFMT_ARGS} .
```
Comment (bltavares) [inline shfmt/entrypoint.sh:17]:
```suggestion
	shfmt -s -l -d ${SHFMT_ARGS:-} .
```

**Comment by bltavares** [comment]
Thank you :)

**Pull request merged**